### DISCUSSION TITLE
Computer Server for MacOS

### DISCUSSION THREAD

**Discussion by HarshGG** [SimDynamX/SpaceTeamsPRO#39] [2024 Lunar SAR Competition Q&A] [answered]
How can I download the compute server on MacOS? I understand that I cannot run the sim on MacOS, but the system requirements say I can run Python on the compute server.

Thank you!

**Comment by connorjak** [comment, marked as answer]
We don't currently release public releases on MacOS or Linux. We will eventually have this, but it will most likely not be available before the end of the competition. For now, STPRO is Windows 10 or 11 only.

https://github.com/SimDynamX/SpaceTeamsPro/wiki/System-Requirements#compatibility-compute_server--python-api-only

Furthermore, there are some more obstacles for getting the SpaceCRAFT unreal client application (sim graphics) running on Linux and MacOS. Graphics support for MacOS and Linux will come after the basic Compute_Server / Python API support. The STU Lunar SAR competition heavily relies on SpaceCRAFT for features such as scanning simulation.

Some people have had mixed success using Parallels (in game-optimized mode with expanded performance settings) to run an earlier version of Space Teams on their computers.
